nifi-commons/nifi-expression-language/pom.xml

```diff
@@ -108,5 +108,10 @@
             <artifactId>commons-codec</artifactId>
             <version>1.14</version>
         </dependency>
+        <dependency>
+            <groupId>ch.hsr</groupId>
+            <artifactId>geohash</artifactId>
+            <version>1.4.0</version>
```

Comment:
Version 1.5.0 now available, should we use that?

Also, it's Apache 2.0 licensed (great), but presumably we need to update our LICENSE/NOTICE files to reflect the fact we're using the library?

Reply:
Like you've done for https://github.com/apache/nifi/pull/5476/files

Reply:
There shouldn't actually be a need to include it in the LICENSE/NOTICE file. The library is licensed as Apache License 2.0 and does not provide a NOTICE file.

Reply:
Please see [here](https://github.com/apache/nifi/pull/5476#discussion_r764415370) regarding the version update. An additional notes for licensing: the NOTICE information is the same for [NIFI-9333](https://github.com/apache/nifi/pull/5518) and [NIFI-9259](https://github.com/apache/nifi/pull/5476), so NIFI-9333 can be merged after NIFI-9259 so that the information is included.